src/libraries/System.Runtime/tests/System/EnumTests.cs

```diff
@@ -2436,19 +2336,6 @@ private static EnumBuilder GetNonRuntimeEnumTypeBuilder(Type underlyingType)
             return module.DefineEnum("TestName_" + underlyingType.Name, TypeAttributes.Public, underlyingType);
         }
 
-        private static Type s_boolEnumType = GetBoolEnumType();
-        private static Type GetBoolEnumType()
-        {
-            EnumBuilder enumBuilder = GetNonRuntimeEnumTypeBuilder(typeof(bool));
```

Comment:
Ideally we'd also have a test that trying to create an enum with bool as the underlying type fails.